templates/project/.php-cs-fixer.dist.php

```diff
@@ -64,7 +64,6 @@
     'static_lambda' => true,
     'strict_comparison' => true,
     'strict_param' => true,
-    'types_spaces' => ['space' => 'single'],
```

Comment:
Types_spaces is configured to 'none' by the Symfony standard and we tried to not override the default standard so far.